apps/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -706,6 +708,14 @@ export class RushConfiguration {
     return this._ensureConsistentVersions;
   }
 
+  /**
+   * If true, inter-package dependencies will be stored as '*' in package.json
+   * but emitted as exact versions during publishing.
+   */
+  public get storeLooseVersions(): boolean {
```

Comment:
I am very open to schema suggestions here. Should this be a version policy? A sub-object specifying both the 'stored' version behavior and the 'emitted' version behavior? Ideas welcome.

Reply:
I think this should be in a version policy. It doesn't seem right to put it directly in `rush.json`

Reply:
I will work on making this part of version policy. It occurred to me that I could factor this into some sort of pair of `setDependencyVersionForPublishing` and `storeDependencyVersionInSource` on the `VersionPolicy` class, so the entire concept, including implementation, could be delegated to version policy.